src/repr/src/row/encoding.rs

```diff
@@ -439,6 +431,71 @@ impl DatumToPersist for NullableProtoDatumToPersist {
     }
 }
 
+/// An implementation of [DatumToPersist] that maps to/from all non-nullable
+/// Datum types using the ProtoDatum representation and doesn't collect any
+/// stats.
+///
+/// Note: This and the other `ProtoDatumToPersist*` exist as temporary
+/// placeholders while we work to nail down the best Arrow representations for
+/// Datum.
+#[derive(Debug)]
+pub struct ProtoDatumToPersistNoStats;
```

Comment:
Note: I added a new type instead of changing the existing `ProtoDatumToPersist` type to punt on changing the stored stats for a type.